priv/repo/migrations/20170427090331_create_class.exs

```diff
@@ -0,0 +1,18 @@
+defmodule CoursePlanner.Repo.Migrations.CreateClass do
+  use Ecto.Migration
+
+  def change do
+    create table(:classes) do
+      add :class_date, :date
```

Comment:
what do you think of renaming this to `date`? this is already part of the class table